hazelcast/src/main/java/com/hazelcast/config/DiscoveryConfig.java

```diff
@@ -135,7 +142,7 @@ public String toString() {
         return "DiscoveryConfig{"
                 + "discoveryStrategyConfigs=" + discoveryStrategyConfigs
                 + ", discoveryServiceProvider=" + discoveryServiceProvider
-                + ", nodeFilter=" + nodeFilter
+                + ", nodeFilter=" + nodeFilter + '\''
```

Comment:
Single quotation marks are not needed on line 145. We use it only to escape class names which are strings. The same problem appears in other classes.